app/src/pages/inside/common/infoLine/infoLineListView.jsx

```diff
@@ -15,58 +15,77 @@
  */
 
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
 import { injectIntl, defineMessages } from 'react-intl';
-import { SharedFilterIcon } from 'pages/inside/common/sharedFilterIcon';
+import {
+  queryParametersSelector,
+  namespaceSelector,
+  COMPOSITE_ATTRIBUTES_FILTER,
+} from 'controllers/testItem';
 import styles from './infoLine.scss';
-import { Owner } from './owner';
-import { Description } from './description';
 
 const cx = classNames.bind(styles);
 const messages = defineMessages({
   filter: {
     id: 'InfoLineListView.filter',
-    defaultMessage: 'Filter',
+    defaultMessage: 'Filter:',
+  },
+  filteredBy: {
+    id: 'InfoLineListView.filteredBy',
+    defaultMessage: 'Filtered by:',
   },
 });
 
+@connect((state) => ({
+  queryParameters: queryParametersSelector(state, namespaceSelector(state)),
+}))
 @injectIntl
 export class InfoLineListView extends Component {
   static propTypes = {
-    data: PropTypes.object,
+    queryParameters: PropTypes.object.isRequired,
     intl: PropTypes.object.isRequired,
-    currentUser: PropTypes.string,
+    data: PropTypes.object,
   };
   static defaultProps = {
     currentUser: '',
     data: {},
   };
 
   render() {
-    const { data, currentUser } = this.props;
-    const { formatMessage } = this.props.intl;
+    const {
+      data,
+      queryParameters,
+      intl: { formatMessage },
+    } = this.props;
+    const compositeAttributes = queryParameters[COMPOSITE_ATTRIBUTES_FILTER];
```

Comment:
I suggest to create a separate selector for this purpose: `compositeAttributesSelector`.